src/components/_app/useKeyPairContext.tsx

```diff
@@ -431,5 +702,7 @@ const useKeyPair = () => {
   }
 }
 
-export { deleteKeyPairFromIdb, getKeyPairFromIdb, setKeyPairToIdb }
+const useKeyPair = () => useContext(KeyPairContext)
+
+export { KeyPairProvider, deleteKeyPairFromIdb, getKeyPairFromIdb }
 export default useKeyPair
```

Comment:
We can remove the default export and just use named export, just like we do it in every other `...Context` components:

```ts
export { KeyPairProvider, useKeyPair, deleteKeyPairFromIdb, getKeyPairFromIdb }
```